client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -2,52 +2,85 @@
 /**
  * External dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
-import { SelectControl, TextControl, Button } from 'newspack-components';
+import { Button, CheckboxControl } from 'newspack-components';
+import { includes, filter } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { H, Card } from '@woocommerce/components';
 
 class Industry extends Component {
 	constructor() {
 		super();
 		this.state = {
-			textInput: '',
-			selectValue1: '1st',
-			selectValue2: '',
+			selected: [],
 		};
+		this.onContinue = this.onContinue.bind( this );
+		this.onChange = this.onChange.bind( this );
+	}
+
+	async onContinue() {
+		const updateProfile = await this.props.updateProfile( { industry: this.state.selected } );
+		if ( updateProfile && 'success' === updateProfile.status ) {
+			this.props.goToNextStep();
+		}
 	}
+
+	onChange( slug ) {
+		this.setState( state => {
+			if ( includes( state.selected, slug ) ) {
+				return {
+					selected:
+						filter( state.selected, value => {
+							return value !== slug;
+						} ) || [],
+				};
+			}
+			const newSelected = state.selected;
+			newSelected.push( slug );
+			return {
+				selected: newSelected,
+			};
+		} );
+	}
+
 	render() {
+		const { industries } = wcSettings.onboarding;
+		const { selected } = this.state;
 		return (
 			<Fragment>
-				<TextControl
-					label={ 'Text Input' }
-					value={ this.state.textInput }
-					onChange={ value => {
-						this.setState( { textInput: value } );
-					} }
-				/>
-
-				<SelectControl
-					label="Select with value"
-					value={ this.state.selectValue1 }
-					options={ [
-						{ value: '1st', label: 'First' },
-						{ value: '2nd', label: 'Second' },
-						{ value: '3rd', label: 'Third' },
-					] }
-					onChange={ value => this.setState( { selectValue1: value } ) }
-				/>
-
-				<SelectControl
-					label="Select empty"
-					value={ this.state.selectValue2 }
-					options={ [
-						{ value: '1st', label: 'First' },
-						{ value: '2nd', label: 'Second' },
-						{ value: '3rd', label: 'Third' },
-					] }
-					onChange={ value => this.setState( { selectValue2: value } ) }
-				/>
+				<H className="woocommerce-profile-wizard__header-title">
+					{ __( 'In which industry does the store operate?', 'woocommerce-admin' ) }
+				</H>
+				<p>{ __( 'Choose any that apply' ) }</p>
+				<Card className="woocommerce-profile-wizard__industry-card">
+					<div className="woocommerce-profile-wizard__checkbox-group">
+						{ Object.keys( industries ).map( slug => {
+							return (
+								<CheckboxControl
+									key={ slug }
+									label={ __( industries[ slug ], 'woocommerce-admin' ) }
```

Comment:
I think these are already localized in the API, so we might not need to localize these strings.

Reply:
Good catch 👍. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2323/commits/45ec30ad54b2cb7da99735b94959a350714f5040.